File: benchmark-java-node-php-python/php-laravel/monolith/app/Models/Bonus.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Bonus extends Model
{
    protected $table = 'bonus';
    protected $fillable = ['amount', 'description', 'client_id', 'expiration_date', 'created_at'];
    public $timestamps = false;

    public function toArray()
    {
        return [
            'id' => $this->id,
            'amount' => (float) $this->amount,
            'description' => $this->description,
            'clientId' => $this->client_id,
            'createdAt' => $this->created_at,
            'expirationDate' => $this->expiration_date,
        ];
    }
}

File: benchmark-stacks-swarm/stacks/apps/python-fastapi/app/models/bonus.py
from sqlalchemy import Column, Integer, Numeric, String, DateTime
from datetime import datetime
from app.core.database import Base

class Bonus(Base):
    __tablename__ = "bonus"
    id = Column(Integer, primary_key=True, index=True)
    amount = Column(Numeric)
    description = Column(String)
    clientId = Column("client_id", String)
    createdAt = Column("created_at", DateTime, default=datetime.utcnow)
    expirationDate = Column("expiration_date", DateTime)
